hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/namenode/ha/TestEditLogTailer.java

```diff
@@ -429,19 +431,31 @@ public void testStandbyTriggersLogRollsWhenTailInProgressEdits()
       waitForStandbyToCatchUpWithInProgressEdits(standby, activeTxId,
           standbyCatchupWaitTime);
 
+      long curTime = standby.getNamesystem().getEditLogTailer().getTimer()
+          .monotonicNow();
+      long inSufficientTimeForLogRoll =
+          TimeUnit.SECONDS.toMillis(logRollPeriod / 3);
```

Comment:
A few places changing this `logRollPeriod` from seconds to milliseconds. It seems clearer to jut rename it to `logRollPeriodMs` and use values in milliseconds.